### PR TITLE
Add testimonial and logo for Scrive

### PR DESCRIPTION
**Pull request by theophile-scrive**
This PR adds an entry in the testimonials for my employer Scrive AB. 

cc @tageborg

### CODE REVIEW

**Review by tomjaguarpaw — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
CI says

```
AesonException "Error in $: parsing Testimonial.Testimonial(Testimonial) failed, key \"shortTestimonial\" not found"
```

File: site/testimonials/scrive.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+companyName: Scrive
+companyURL: https://scrive.com
+logoURL: logos/scrive.png
```
Comment (tomjaguarpaw) [inline site/testimonials/scrive.yaml:4]: Maybe it needs this now?

```suggestion
logoURL: logos/Scrive.png
```

**Comment by theophile-scrive** [comment]
Thanks for merging! I now realise that the card isn't the same size at the others: 
image

Is there a fixed logo size required?

**Comment by tomjaguarpaw** [comment]
The others do seem to be 300px tall.

**Pull request merged**